client/src/Account/components/AccountBalancePieChart.tsx

```diff
@@ -38,39 +38,24 @@ const AccountBalancePieChart: FC<Props> = ({ account }) => {
     },
   ]
 
-  const emptyState = [{ id: 'No value to show', label: '', value: true, color: '#e5e7eb' }]
+  const emptyState = [{ id: 'No value to show', label: '', value: 0, color: '#e5e7eb' }]
 
   const isEmpty = other === 0 && staking === 0 && transferable === 0
 
   return (
     <div className='sm:h-80 sm:w-2/4 h-60 w-full'>
-      {isEmpty ? (
-        <ResponsivePie
-          data={emptyState}
-          margin={{ top: 20, right: 0, bottom: 40, left: 0 }}
-          innerRadius={0.5}
-          padAngle={0}
-          cornerRadius={3}
-          activeOuterRadiusOffset={8}
-          colors={{ datum: 'data.color' }}
-          enableArcLabels={false}
-          sortByValue={true}
-          enableArcLinkLabels={true}
-        />
-      ) : (
-        <ResponsivePie
-          data={data}
-          margin={{ top: 20, right: 0, bottom: 40, left: 0 }}
-          innerRadius={0.5}
-          padAngle={0}
-          cornerRadius={3}
-          activeOuterRadiusOffset={8}
-          colors={{ datum: 'data.color' }}
-          enableArcLabels={false}
-          sortByValue={true}
-          enableArcLinkLabels={false}
-        />
-      )}
+      <ResponsivePie
```

Comment:
Currently, the empty version of the chart is not being shown:

Current implementation:
![image](https://user-images.githubusercontent.com/5139220/220648917-7f694470-9700-4b15-b60a-86193d7b5e5e.png)

How it should show the empty state:
![image](https://user-images.githubusercontent.com/5139220/220649140-c620acc8-54e6-4766-a078-712e189eb52a.png)